src/nifs/sangria/accumulator.rs

```diff
@@ -219,7 +300,11 @@ where
 {
     pub fn new(args: RelaxedPlonkTraceArgs) -> Self {
         Self {
-            U: RelaxedPlonkInstance::new(args.num_challenges, args.num_witness),
+            U: RelaxedPlonkInstance::new(
+                args.num_challenges,
+                args.num_witness,
+                args.num_io.len() - 1, // instances
```

Comment:
why is `args.num_io.len()-1` here?

Reply:
Because this fn arg is number of step circuit instances, not all instances len

Reply:
so the remain one is the accumulator instance?